test_utils/factories.py

```diff
@@ -116,6 +116,43 @@ class Meta:
     subcategory = factory.SubFactory(SkillSubCategoryFactory)
 
 
+# pylint: disable=no-member
+class XblockSkillsFactory(factory.django.DjangoModelFactory):
```

Comment:
Nit: I think `XBlock` is more consistent capitalization with the rest of the platform, not `Xblock`

Reply:
Makes sense. I'll update.